test/apparmor.bats

```diff
@@ -14,6 +14,9 @@ function teardown() {
     if [[ "$enabled" -eq 0 ]]; then
         skip "skip this test since apparmor is not enabled."
     fi
+    if [[ "$CI" == "true" ]]; then
```

Comment:
Can we add a comment why that's needed? It might be the heat but it's not obvious to me why it's needed now.

Reply:
Hm, I never got this managed to work because of mainly this: https://stackoverflow.com/questions/25533666/cannot-reload-or-start-apparmor-in-docker/27029610#27029610

The tests are host dependent, so It's not nice to run them in parallel even if the host supports apparmor.

I will add a comment about that.

Reply:
Docker uses a trick. There's a `dind` (Docker in Docker) somewhere in the tree to setup the mounts and privileges that AppArmor requires to be set.